src/me/remie/vulcan/leaguetasks/utils/LeaguePanel.java

```diff
@@ -0,0 +1,119 @@
+        return false;
+    }
+
+    private boolean isOptionSelected(SimpleWidget widget, int childIndex, String option) {
+        if (widget == null || widget.getChild(childIndex) == null) return false;
+        return widget.getChild(childIndex).getText().equalsIgnoreCase(option);
+    }
+
+    public boolean clearAllFilterOptions() {
+        if (!isLeaguePanelOpen()) return false;
+        final SimpleWidget tierDropDownMenu = ctx.widgets.getWidget(657,27);
+        if (tierDropDownMenu != null) {
+            if (!isOptionSelected(tierDropDownMenu, 4, "all")) {
+                final SimpleWidget selectedTierOption = tierDropDownMenu.getChild(4);
+                final SimpleWidget dropDownMenu = ctx.widgets.getWidget(657, 35);
+                if (!chooseOptionInDropdown(selectedTierOption, dropDownMenu,1)) {
+                    ctx.log("Something went wrong choosing tier option");
+                    return false;
+                }
+            }
+        }
+
+        final SimpleWidget typeDropDownMenu = ctx.widgets.getWidget(657,28);
+        if (typeDropDownMenu != null) {
+            if (!isOptionSelected(typeDropDownMenu, 4, "all")) {
+                final SimpleWidget selectedTierOption = typeDropDownMenu.getChild(4);
+                final SimpleWidget dropDownMenu = ctx.widgets.getWidget(657, 36);
+                if (!chooseOptionInDropdown(selectedTierOption, dropDownMenu,1)) {
+                    ctx.log("Something went wrong choosing type option");
+                    return false;
+                }
+            }
+        }
+
+        final SimpleWidget areaDropDownMenu = ctx.widgets.getWidget(657,29);
+        if (areaDropDownMenu != null) {
+            if (!isOptionSelected(areaDropDownMenu, 4, "all")) {
+                final SimpleWidget selectedTierOption = areaDropDownMenu.getChild(4);
+                final SimpleWidget dropDownMenu = ctx.widgets.getWidget(657, 37);
+                if (!chooseOptionInDropdown(selectedTierOption, dropDownMenu,1)) {
+                    ctx.log("Something went wrong choosing area option");
+                    return false;
+                }
+            }
+        }
+
+        final SimpleWidget completedDropDownMenu = ctx.widgets.getWidget(657,31);
+        if (completedDropDownMenu != null) {
+            if (!isOptionSelected(completedDropDownMenu, 4, "all")) {
+                final SimpleWidget selectedTierOption = completedDropDownMenu.getChild(4);
+                final SimpleWidget dropDownMenu = ctx.widgets.getWidget(657, 39);
+                if (!chooseOptionInDropdown(selectedTierOption, dropDownMenu,1)) {
+                    ctx.log("Something went wrong choosing completed option");
+                    return false;
+                }
+            }
+        }
+        return true;
+    }
+
+    private boolean chooseOptionInDropdown(SimpleWidget dropDownWidget, SimpleWidget dropDownOptionList, int option) {
```

Comment:
```suggestion
    private boolean chooseOptionInDropdown(SimpleWidget dropDownWidget, SimpleWidget dropDownOptionList) {
```
We don't need the option parameter as I only ever see it being passed with the value: `1`, we can just hard code it inside of the function.